src/cbor.zig

```diff
@@ -395,18 +446,40 @@ fn additionalInfo(data: []const u8, l: ?*usize) ?u64 {
             if (l != null) l.?.* = 9;
             return @as(u64, @intCast(unsigned_64(data[1..9])));
         },
+        0x1f => {
+            if (data.len < 1) return null;
+            if (l != null) l.?.* = 1;
+            return 0x00;
+        },
         else => return null,
     }
 }
 
+fn valid_indefinite(data: []const u8, i: *usize, mt: u8, breakable: bool) bool {
+    if (i.* >= data.len) return false;
+    switch (mt) {
+        2, 3 => {
+            while (validate(data, i, false, true)) {
+                if (data[i.*] != mt) return false;
+            }
```

Comment:
- Please add a comment explaining that a indefinite string consists of definite length items of the same type. This would make it much easier to understand what the loop does.
- I don't think that you can compare the `data[i.*]` byte with `mt` as the byte is the concatenation of `MT || AI`. Try to extract the major type from the byte before the comparison. Maybe it would also be good to add this line from the spec: `If any item between the indefinite-length string indicator (0b010_11111 or 0b011_11111) and the "break" stop code is not a definite-length string item of the same major type, the string is not well-formed.`

Reply:
I suppose for now I have only explicitly supported indefinite length arrays, and not strings. Perhaps I should make indefinite length strings explicitly unsupported for now, as I don't need them for my use case.